src/store/whiteboarding/fabric-utils.ts

```diff
@@ -890,10 +893,10 @@ const $_initSocket = (state: any) => {
     transports: ['websocket', 'polling'],
     withCredentials: true
   })
-  p.$socket.on('close', $_tryReconnect)
+  p.$socket.on('close', () => $_tryReconnect(state))
```

Comment:
The line altered above was problematic. It would not pass valid 'state' object. Not sure if this potential bug actually manifested.